<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">
        <property name="windowModality">
            <enum>Qt::ApplicationModal</enum>
        </property>
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>773</width>
                <height>455</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Historia Clínica</string>
        </property>
        <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <widget class="QLabel" name="logo">
            <property name="geometry">
                <rect>
                    <x>60</x>
                    <y>20</y>
                    <width>141</width>
                    <height>91</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">background-color: rgb(170, 0, 0);</string>
            </property>
            <property name="pixmap">
                <pixmap>../assets/logo_optica.png</pixmap>
            </property>
            <property name="scaledContents">
                <bool>true</bool>
            </property>
        </widget>
        <widget class="QLabel" name="lblTitle">
            <property name="geometry">
                <rect>
                    <x>239</x>
                    <y>30</y>
                    <width>441</width>
                    <height>60</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>18</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
                <string>Historia Clínica</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLabel" name="lblDocumentType">
            <property name="geometry">
                <rect>
                    <x>60</x>
                    <y>140</y>
                    <width>300</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
                <string>Tipo de documento</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QComboBox" name="txtDocumentType">
            <property name="geometry">
                <rect>
                    <x>79</x>
                    <y>190</y>
                    <width>261</width>
                    <height>31</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    border: 1px solid rgb(0, 0, 0);
                    color: rgb(0, 0, 0);
                    background-color: rgb(255, 255, 255);
                    text-align: center;
                </string>
            </property>
        </widget>
        <widget class="QLabel" name="lblDocumentNumber">
            <property name="geometry">
                <rect>
                    <x>420</x>
                    <y>140</y>
                    <width>300</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
                <string>Número de documento</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLineEdit" name="txtDocumentNumber">
            <property name="geometry">
                <rect>
                    <x>429</x>
                    <y>190</y>
                    <width>271</width>
                    <height>31</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    border: 1px solid rgb(0, 0, 0);
                    color: rgb(0, 0, 0);
                    background-color: rgb(255, 255, 255);
                </string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QTableWidget" name="tableFiles">
            <property name="geometry">
                <rect>
                    <x>80</x>
                    <y>260</y>
                    <width>621</width>
                    <height>61</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="cursor" stdset="0">
                <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    background-color: rgb(255, 255, 255);
                    color: rgb(0, 0, 0);
                </string>
            </property>
            <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustToContents</enum>
            </property>
            <property name="columnCount">
                <number>4</number>
            </property>
            <property name="horizontalHeaderLabels" stdset="0">
                <stringlist>
                    <string>Nombre completo</string>
                    <string>Fecha de radicación</string>
                    <string>Fecha de actualización</string>
                    <string>Descargar</string>
                </stringlist>
            </property>
            <column />
            <column />
            <column />
            <column />
        </widget>
        <widget class="QPushButton" name="btnSearch">
            <property name="geometry">
                <rect>
                    <x>280</x>
                    <y>340</y>
                    <width>221</width>
                    <height>40</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    background-color: rgb(111, 166, 80);
                    color: white;
                    border: 1px solid rgb(0, 0, 0);
                    border-radius: 12px;
                    padding: 10px;
                </string>
            </property>
            <property name="text">
                <string>Buscar</string>
            </property>
        </widget>
        <widget class="QLabel" name="lblError">
            <property name="geometry">
                <rect>
                    <x>70</x>
                    <y>390</y>
                    <width>631</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>11</pointsize>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">color: red;</string>
            </property>
            <property name="text">
                <string>Error</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
    </widget>
    <resources>
        <include location="../logo.qrc" />
        <include location="logo" />
    </resources>
    <connections />
</ui>